### PR TITLE
Added extension method for IApplicationBuilder

### PR DESCRIPTION
**Pull request by ibarban**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Installing this package to a Rhetos web application:
 
 1. Add 'Rhetos.Impersonation' NuGet package, available at the [NuGet.org](https://www.nuget.org/) on-line gallery:
 2. Extend Rhetos services configuration (at `services.AddRhetos`) with the impersonation service: `.AddImpersonation()`
+3. Extend the endpoints with new routes: `endpoints.MapRhetosImperosnation`
```
Comment (bantolov) [inline Readme.md]: Please check if the instructions should be updated. "Use"?

File: src/Rhetos.Host.AspNet.Impersonation/ImperosnationControllerRouteConvention.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+using Microsoft.AspNetCore.Mvc.ApplicationModels;
+using Microsoft.Extensions.Options;
+
+namespace Rhetos.Host.AspNet.Impersonation
+{
+    internal class ImperosnationControllerRouteConvention : IControllerModelConvention
+    {
+        private readonly IOptions<ImpersonationOptions> impersonationOptions;
+
+        public ImperosnationControllerRouteConvention(IOptions<ImpersonationOptions> impersonationOptions)
```
Comment (bantolov) [inline src/Rhetos.Host.AspNet.Impersonation/ImperosnationControllerRouteConvention.cs]: Typo (also check in other files).

File: src/Rhetos.Host.AspNet.Impersonation/ImpersonationOptions.cs [hunk 3]
```diff
@@ -24,5 +24,9 @@ public class ImpersonationOptions
         public const string SectionName = "Impersonation";
 
         public int CookieDurationMinutes { get; set; } = 60;
+
+        public string BaseRoute { get; set; } = "rest/Common";
```
Comment (bantolov) [inline src/Rhetos.Host.AspNet.Impersonation/ImpersonationOptions.cs:31]: Please add a short summary that includes "The default value is backward compatible with older versions of Impersonation plugin."

**Review by bantolov — CHANGES_REQUESTED** [review]

**Review by sstublic — APPROVED** [review]
Looks good.

**Review by bantolov — APPROVED** [review]

**Pull request merged**